cmd/snap/cmd_snap_op.go

```diff
@@ -120,9 +120,10 @@ and the snap can easily be enabled again.
 type cmdRemove struct {
 	waitMixin
 
-	Revision   string `long:"revision"`
-	Purge      bool   `long:"purge"`
-	Positional struct {
+	Revision      string `long:"revision"`
+	Purge         bool   `long:"purge"`
+	Transactional bool   `long:"transactional"`
```

Comment:
I wonder if this shouldn't be named "transactional-all" or something like that since snapd already promises to do things "transactionally", they are just transactional per-snap only for these ops?

Reply:
Actually I don't think we can support that with snap removal? We cannot restore a snap that was removed if the other one fails to remove.

Reply:
How about just `--transaction` ?

Reply:
I am not opposed :-). I'll let @pedronis comment on what would be the best naming.